### DISCUSSION TITLE
Issue on Score and Timer for Soccer games

### DISCUSSION THREAD

**Discussion by ktalovmorx** [cloudbet/docs#14] [Q&A]
Soccer games are not showing time or team scores. How are you supposed to bet without knowing the playing time or game score?
Example...
![image](https://user-images.githubusercontent.com/56266060/139326658-ef07206b-6537-4c3d-9b89-c9d314b059d9.png)

Not only for this event, it's for all events and sometimes works well....

**Comment by kgravenreuth** [comment]
Hi @ktalovmorx ,
Thanks for raising this issue.

We are currently looking into providing time and team scores for more events on our website. We will let you know once we have any more updates on this.

**Reply by footytrade2021** [reply]
How do i get inplay team scores and time from the cloudbet api response? @kgravenreuth  @ktalovmorx @lukehoersten @dlackty @joelr

**Reply by kgravenreuth** [reply]
Hi @footytrade2021,

Apologies for the delay in reply. 

As mentioned in [this discussion](https://github.com/Cloudbet/docs/discussions/19), we can provide you information about how your API bets have resulted via the [Bet History](https://www.cloudbet.com/api/?utm_source=github.com&utm_medium=community&utm_campaign=axis&urls.primaryName=Trading#/Trading/BetHistory) endpoint of our Trading API. 

Note that you need to be registered as a Cloudbet player to use the Trading API key with a minimum of 10 euro deposit as mentioned in our [Trading API blog article](https://www.cloudbet.com/en/blog/posts/sports-betting-api-tutorial-with-golang#toc-0?utm_source=github.com&utm_medium=community&utm_campaign=axis)

For example, If you'd like to know the result of a given event, then you can place bets on the Winner or Moneyline markets for that event and then check the bet results to know the event's results.

We are currently not planning to add results or scores to the Feed API, while we thank you for your suggestions above.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
